File: app/src/main/java/com/snaptiongame/app/data/models/OnboardingInfo.java
package com.snaptiongame.app.data.models;

/**
 * @author Tyler Wong
 */

public class OnboardingInfo {
    public int titleId;
    public int descriptionId;
    public int animationId;

    public static final String TITLE_ID = "title";
    public static final String DESCRIPTION_ID = "description";
    public static final String ANIMATION = "animation";

    public OnboardingInfo(int titleId, int descriptionId, int animationId) {
        this.titleId = titleId;
        this.descriptionId = descriptionId;
        this.animationId = animationId;
    }
}

File: app/src/main/java/com/snaptiongame/app/data/models/OnboardingInfo.kt
package com.snaptiongame.app.data.models

/**
 * @author Tyler Wong
 */

data class OnboardingInfo(val titleId: Int, val descriptionId: Int, val animationId: Int) {
    companion object {
        const val TITLE_ID = "title"
        const val DESCRIPTION_ID = "description"
        const val ANIMATION = "animation"
    }
}
